specification/batch/data-plane/Microsoft.Batch/stable/2018-12-01.8.0/BatchService.json

```diff
@@ -11836,11 +11836,12 @@
           "type": "integer",
           "format": "int32",
           "title": "The maximum number of tasks that can run concurrently on a single compute node in the pool.",
-          "description": "The default value is 1. The maximum value of this setting depends on the size of the compute nodes in the pool (the vmSize setting)."
+          "description": "The default value is 1. The maximum value is the smaller of 4 times the number of cores of the vmSize of the pool or 256."
```

Comment:
Just realized how many 'of x' are in this sentence any way to simplify?

Reply:
Yeah I realized it when writing it too but not sure how to improve: "4 * cores of the vm size of the pool" which is the main culprit